src/styles/preflight-edited.stylus

```diff
@@ -0,0 +1,435 @@
+.fq-base {
+
+  /*
+    1. Use the user's configured `sans` font-family by default.
+  */
+  
+  & {
+    font-family: theme('fontFamily.sans', ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji") !important;
+  } 
+
```

Comment:
I think we could add here annotation that te content below is copied from 
`node_modules/tailwindcss/lib/css/preflight.css`

Reply:
That is half true ;). But we can add that it was base for that file.

That was copy and modyfied to work with prefixed class. 
There was need to add "&" in multiple places. Also removed some part like body and html.